src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -981,6 +981,11 @@ public Diagnostic RuntimeValueNotAllowedInProperty(string propertyName, string?
                 "BCP162",
                 "Expected a loop item variable identifier or \"(\" at this location.");
 
+            public ErrorDiagnostic ExpectedLoopIndexIdentifier() => new(
```

Comment:
>ExpectedLoopIndexIdentifier [](http://example.com/codeflow?start=35&length=27)

Could this be a merge error? The main branch doesn't have this diagnostic because it was removed.